internal/scorecard/alpha/tar.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// tarrer walks paths to create tar file tarName
+func Tartar(tarName string, paths []string) (err error) {
```

Comment:
WDYT about improve the name and the comment to make more clear what it does and its purpose. Why / where the tar is used? Will it creates a tar file with all files found in the paths? May BuildTarWith?